src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/MerchandiseReturnController.php

```diff
@@ -27,17 +27,27 @@
 namespace PrestaShopBundle\Controller\Admin\Sell\CustomerService;
 
 use Exception;
+use PrestaShop\PrestaShop\Adapter\PDF\OrderReturnPdfGenerator;
```

Comment:
The `OrderReturnPdfGenerator` should have an interface inside the core and being inject here instead of the adapter.

The purpose is to have interfaces inside the core, and their implementations inside the adapter namespace. So we can migrate every service from adapters to the core.